index.js

```diff
@@ -219,6 +219,23 @@ function focusTrap(element, userOptions) {
     return node;
   }
 
+  function getReturnFocusNode(previousActiveElement) {
+    var node;
+    if (getNodeForOption('setReturnFocus') !== null) {
+      node = getNodeForOption('setReturnFocus');
+    } else {
+      node = previousActiveElement;
+    }
+
+    if (!node) {
+      throw new Error(
+        "You can't have a focus-trap without at least one focusable element"
```

Comment:
Badly worded if I'm correct, probably better would be to avoid double negation here like:

```suggestion
        "Your focus-trap needs to have at least one focusable element"
```